packages/application-generic/src/factories/sms/sms.factory.ts

```diff
@@ -19,6 +19,8 @@ import {
   AfricasTalkingSmsHandler,
   SendchampSmsHandler,
 } from './handlers';
+import { SmsProviderIdEnum } from '@novu/shared';
+import { NovuSmsHandler } from './handlers/novu.handler';
```

Comment:
You can move the `NovuSmsHandler` to the `./handlers` import of line 21 and above.